src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name, hidden = false) {
+    super(health, name);
```

Comment:
The order of parameters in the Herbivore constructor is incorrect. According to the task description, 'name' should come before 'health'. Additionally, the 'super' call should match the order of parameters in the Animal constructor, which is 'name' followed by 'health'.